oml/models/vit_unicom/extractor.py

```diff
@@ -0,0 +1,95 @@
+from pathlib import Path
+from typing import Optional, Union
+
+import torch
+
+from oml.const import CKPT_SAVE_ROOT
+from oml.interfaces.models import IExtractor
+from oml.models.utils import remove_prefix_from_state_dict
+from oml.models.vit_unicom.external import vision_transformer
+from oml.models.vit_unicom.external.model import load  # type: ignore
+from oml.utils.misc_torch import normalise
+
+
+def unicom_vitb32() -> vision_transformer.VisionTransformer:  # type: ignore
```

Comment:
Why have you decided to use namespaces instead of objects?

I mean
```python
from oml.models.vit_unicom.external import vision_transformer

vision_transformer.build_model(...)
vision_transformer.VisionTransformer(...)
```

Instead of 
```python
from oml.models.vit_unicom.external.vision_transformer import build_model, VisionTransformer

build_model(...)
VisionTransformer(...)
```

It's ok, but rare usage, especially in OML